.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+
+sudo: false
+
+matrix:
+  include:
+    - python: 3.5
+      env: TOX_ENV='flake8'
+    - python: 3.5
+      env: TOX_ENV='isort'
+    # Django 2.0
+    - python: 3.5
+      env: DJANGO='dj20' CMS='cms36'
```

Comment:
Please add more jobs here for other python versions supported in Django 2.0
https://docs.djangoproject.com/en/2.1/releases/2.0/#python-compatibility